### DISCUSSION TITLE
Trigger fzf autocomplete even if next character is not a space.

### DISCUSSION THREAD

**Discussion by yrammos** [Aloxaf/fzf-tab#404] [Q&A]
I would expect fzf-tab to work analogously with ZSH’s `expand-or-complete-prefix`, for example:

```
<TAB>edit
```

would suggest `sudoedit` as a possible completion.

Currently, fzf-tab fails to work in such situations.

Thanks in advance for any insights!

**Comment by Aloxaf** [comment]
Umm, AFAIK, what `expand-or-complete-prefix` do is ignoring suffix.
